tests/test_other.py

```diff
@@ -5251,7 +5253,7 @@ def test_locale_wrong(self):
     ''')
     run_process([PYTHON, EMCC, 'src.cpp', '-s', 'EXIT_RUNTIME=1', '-s', 'DISABLE_EXCEPTION_CATCHING=0'])
     self.assertContained('Constructed locale "C"\nThis locale is the global locale.\nThis locale is the C locale.', run_js('a.out.js', args=['C']))
-    self.assertContained('''Can't construct locale "waka": collate_byname<char>::collate_byname failed to construct for waka''', run_js('a.out.js', args=['waka'], assert_returncode=1))
+    self.assertContained('Can\'t construct locale "waka": collate_byname<char>::collate_byname failed to construct for waka', run_js('a.out.js', args=['waka'], assert_returncode=1))
```

Comment:
I think triple quotes are better than escaping, but I don't feel strongly, up to you.

Reply:
Hmm.. maybe r"" is better then.   I usually reserve triple quotes for multi-line strings.